### DISCUSSION TITLE
i cant reverify my academic affiliation

### DISCUSSION THREAD

**Discussion by Mr54233** [community/community#54916] [GitHub Education]
### Hi everyone,

Around March or April, I successfully verified my student status using my email and student information. However, a few days ago (maybe a few weeks ago), GitHub sent me an email asking me to re-verify my status. When I selected my school on the verification webpage, it asked me to use an educational email ending with the school's domain name for verification. However, many Chinese schools do not provide educational emails for students, and the email I used initially was able to pass the verification. Additionally, due to my graduation internship, I am no longer near the school, and any uploads will be questioned for not being in the vicinity of the selected school. I don't know how to proceed, and I can't find any customer service to contact. Can anyone help me?

**Comment by WangMinan** [comment]
Exactly same situation. Dpn't know how to solve it.

**Comment by samus-aran** [comment]
Hi @Mr54233  ! 

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
